tap_linkedin_ads/tap.py

```diff
@@ -71,6 +71,18 @@ class TapLinkedInAds(Tap):
             required=True,
             description="LinkedInAds Owner ID",
         ),
+        th.Property(
+            "campaign_group",
+            th.StringType,
+            required=True,
+            description="LinkedInAds Campaign Group ID",
```

Comment:
Can you expand this description a bit to mention which streams this setting is required for?